Chatlis-client/Source/QChatbox.cpp

```diff
@@ -39,12 +40,16 @@ void QChatbox::appendServerMessage(const QString message)
 void QChatbox::appendUserMessage(const QString& from, const QString& message)
 {
     setTextColor(QApplication::palette().text().color());
+    if (message.contains(QString("@" + clientInfo.getUsername()))) {
```

Comment:
QStringView wouldn't work here because we would be getting a one time value, but the username can change, if we have a reference of the clientInfo, if the client changes their username, the check will update dynamically.